src/components/data-management/LaunchAnalysisButton.jsx

```diff
@@ -113,6 +113,8 @@ const LaunchAnalysisButton = () => {
     };
 
     const canLaunch = activeProject.samples.every((sampleUuid) => {
+      if (!samples[sampleUuid]) return false;
```

Comment:
In which cases is there a sampleId in activeProject but it doesn't show up in samples?

Reply:
In the case of experimentId `fe96449d41b0bac2a9c48dbf9f10db8d`. If you compare the requests that are sent to update the samples record here: https://biomage.atlassian.net/browse/BIOMAGE-1745?focusedCommentId=10813, you can see that in the later update, a `sampleId` goes missing, although the ID exists in the `project.samples`.